test/test_rounding.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+'''
+test_rounding.py - Unit tests for pyfuse3.
+
+Copyright © 2020 Philip Warner <philipwarner.info>
+
+This file is part of pyfuse3. This work may be distributed under
+the terms of the GNU LGPL.
+'''
```

Comment:
some pep8 checkers / syntax highlighters do not like `'''` for docstrings, so better use `"""`-

Reply:
Code was  cloned from existing source file and is consistent with module style; probably best to change all existing source files at the same time...and only if the owner is happy with the style change, but I don't have time for that. Feel free!

Reply:
TBC: I do not plan to fix this since it should be done to all sources if at all, and I have scant time available.

Reply:
if that is consistent with rest of source -> fine!